extensions/patterns/configure_ec2/exec_env/execution-environment.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
 
 images:
   base_image:
-    name: docker.io/redhat/ubi9:latest
+    name: registry.redhat.io/ubi9/ubi:latest
```

Comment:
You can't use this url. You have to use registry.access.redhat.com/ubi9/ubi:latest. The link I posted shows you how to get unauthenticated access to the image.

Reply:
updated